automatic_cross_compile.sh

```diff
@@ -0,0 +1,61 @@
+# This script is intended to be used by the Jenkins night builds.
+# But it still possible to use manually setting up this variables:
+#
+# TARGET=[raspbian]
+# ROS2_DISTRO=[ardent,bouncy,crystal,master,...]
+
```

Comment:
Could be worth adding the command line `set -e` to stop in case an error happens.